From 097d93c71ffe41a046286e65f18fbbc52fdb6831 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Mon, 14 Aug 2017 11:19:19 +0800
Subject: [PATCH] ARM: dts: rk3288-android: enable the nandc node by default

Enable the nand node by default in the android dtsi as
they're wired on every board for drmboot compatible.

Change-Id: I63aea9be6ca43fb91f7ec6616f5b9051ca5c23a8
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-android.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288-android.dtsi b/arch/arm/boot/dts/rk3288-android.dtsi
index c3f1239095c6..a13be09e3d18 100644
--- a/arch/arm/boot/dts/rk3288-android.dtsi
+++ b/arch/arm/boot/dts/rk3288-android.dtsi
@@ -334,7 +334,7 @@
 		nandc_id = <0>;
 		clocks = <&cru SCLK_NANDC0>, <&cru HCLK_NANDC0>;
 		clock-names = "clk_nandc", "hclk_nandc";
-		status = "disabled";
+		status = "okay";
 	};
 
 	hdmi_analog_sound: hdmi-analog-sound {
-- 
2.35.3

